runtime/make/Makefile.runtime.jemalloc-jemalloc

```diff
@@ -0,0 +1,21 @@
+# Copyright 2004-2017 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+RUNTIME_INCLS += -I$(JEMALLOC_INCLUDE_DIR)
+# Chapel third-party install of jemalloc calls the routines
+# chpl_je_malloc, chpl_je_free, etc
+RUNTIME_DEFS += -DCHPL_JEMALLOC_PREFIX=chpl_je_
```

Comment:
That seems to me to be more trouble than it's worth because we also have to work with --with-private-namespace so it's not a simple trade for 1 variable. Anyway it would add a new Makefile variable and I don't know about you but I'm more likely to be confused by one of those than to be confused by needing to change chpl_je_ in two places.